app/presenters/quadicons/quadrants/auth_status.rb

```diff
@@ -0,0 +1,21 @@
+module Quadicons
+  module Quadrants
+    class AuthStatus < Quadrants::Base
+      def path
+        "100/#{h(img)}.png"
```

Comment:
Just keep the whole string together please, regardless of where it lives.

`"x"` is a nonsensical string on it's own, `"100/x.png"` actually has a meaning :).